accumulator/accumulate_signals_activities.go

```diff
@@ -0,0 +1,25 @@
+package accumulator
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"go.temporal.io/sdk/activity"
+)
+
+// this activity will process all of the signals together
+func ComposeGreeting(ctx context.Context, s []AccumulateGreeting) (string, error) {
+	log := activity.GetLogger(ctx)
+	if(len(s) < 1) {
+		log.Warn("No greetings found when trying to Compose Greetings. ")
```

Comment:
```suggestion
		log.Warn("No greetings found when trying to Compose Greetings.")
```

Unnecessary trailing space in log

Reply:
good, thanks, fixed